utils/validate_images.js

```diff
@@ -27,31 +28,52 @@ const validateImageCounts = (quickstartDirs) => {
       const quickstartDirName = path.dirname(quickstart);
       // get all images for a quickstart
       const imagePaths = glob.sync(
-        path.resolve(path.dirname(quickstart), '**/*.+(png|jpeg|jpg|svg)')
+        path.resolve(quickstartDirName, '**/*.+(png|jpeg|jpg|svg)')
       );
       const quickstartConfig = readQuickstartFile(quickstart).contents[0];
       const iconPath = quickstartConfig.icon
-        ? path.resolve(path.dirname(quickstart), quickstartConfig.icon)
+        ? path.resolve(quickstartDirName, quickstartConfig.icon)
         : null;
       const logoPath = quickstartConfig.logo
-        ? path.resolve(path.dirname(quickstart), quickstartConfig.logo)
+        ? path.resolve(quickstartDirName, quickstartConfig.logo)
         : null;
-
+      
+      // Max images is per dashboard so we need to account for this by getting the number of dashboards
+      const dashboardCount = glob.sync(
+        path.resolve(quickstartDirName, 'dashboards/*.json')
+      ).length;
+      
       const screenshotPaths = imagePaths.filter(
-        (p) => p !== iconPath && p !== logoPath
+        (p) => p !== iconPath && p !== logoPath && !p.includes(DASHBOARD_IMAGES_PATH)
```

Comment:
Something to consider here, what happens if we have a quickstart named `image` or `images`?